_pages/en/web-services-access.md

```diff
@@ -1,63 +1,57 @@
 ---
 layout: default
-title: Granting access to Web tools at ESDC
+title: Granting More Open Access to the Internet for IITB Employees
 ref: web-services-access
 lang: en
 status: posted
 sections: Work In Progress
 permalink: /web-services-access.html
 ---
 
-## Granting access to Web tools at ESDC
+## Granting More Open Access to the Internet for IITB Employees
 
-### Purpose
+### Summary
 
-To seek endorsement for granting access to more Web tools or Software as a Service (SaaS) for IITB employees, as a step toward broader open access to the Internet for all ESDC employees.
+Departments are leveraging Web tools or Software as a Service (SaaS) to collaborate and engage with citizens and public administrations both in Canada and across the globe.
+For example, the latest Open Government Plan and Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
+Many teams are also using Slack or GCcollab Message to communicate internally and across departments.
+The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
 
-### Background
+In order to improve ESDC's ability to leverage current Web tools, a more open access to Web tools or Software as a Service (SaaS) for IITB employees, as a step toward broader open access to the Internet for more ESDC employees.
 
-The upcoming [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) replaces the [Policy on Acceptable Network and Device Use (PANDU)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27122) but keeps the intent to "provide open access to Internet tools and services to support public servants in their work duties, enhance collaboration and networking with their peers".
-The policy applies only to unclassified interactions (data, information, code, etc.).
+As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, the risk would be minimal.
+In addition, the benefits should bring great value such as enhancing collaboration and efficiency, increasing job satisfaction and helping attract and retain IT talent.
```

Comment:
Substantiate this with links to DORA's cultural capabilities:
- [Organization Culture](https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture) that favours information flow, team collaboration, and breaking down barriers. 
- [Job Satisfaction](https://cloud.google.com/solutions/devops/devops-culture-job-satisfaction), in particular an identified impediment being not giving people the right tools to be successful